project_redss/lib/dataset_specification.py

```diff
@@ -22,35 +23,49 @@ class DatasetSpecification(object):
                    time_field="sent_on",
                    coda_filename="s01e01",
                    cleaner=None,
-                   code_translator=S01E01Translator),
+                   code_scheme=CodeSchemes.S01E01),
 
         CodingPlan(raw_field="rqa_s01e02_raw",
                    coded_field="rqa_s01_e02_coded",
                    time_field="sent_on",
                    coda_filename="s01e02",
                    cleaner=None,
-                   code_translator=S01E01Translator),  # TODO: Use S01E02 when available
+                   code_scheme=CodeSchemes.S01E01),  # TODO: Use S01E02 when available
```

Comment:
This seems a bit error prone - could we put something in that breaks if we accidentally use this before it's fixed?